osprey_async_worker/Dockerfile

```
@@ -45,10 +45,11 @@ ADD osprey_rpc/pyproject.toml /osprey/osprey_rpc/pyproject.toml
 ADD osprey_worker/pyproject.toml /osprey/osprey_worker/pyproject.toml
 ADD osprey_async_worker/pyproject.toml /osprey/osprey_async_worker/pyproject.toml
 ADD example_plugins/pyproject.toml /osprey/example_plugins/pyproject.toml
+ADD example_atproto_plugins/pyproject.toml /osprey/example_atproto_plugins/pyproject.toml
```

Hadolint (2.15.1)

[error] Use COPY instead of ADD for files and folders

(DL3020)